src/component/Todo.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import TodoForm from '../component/TodoForm';
+import TodoItem from '../component/TodoItem';
+import Todos from '../mock/data';
+
+class Todo extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      id: '',
+      content: '',
+    }
+  }
+
+  // it sets the value on any change
+   onChange = (event) => {
+    event.preventDefault();
+    this.setState({ [event.target.name]: event.target.value, id: Todos.length + 1 })
+  }
+
+  onSubmit = (event) => {
+    event.preventDefault();
+    console.log('@state', this.state);
+    if(this.state.content){
+      Todos.push(this.state);
+      this.setState({id: '', content: ''});
+    }
+    
+    console.log('@todos', Todos);
+  }
+  render() {
+    
+    return (
+      <React.Fragment>
+        <div className="item">
+          <div className="items">
+            <TodoForm 
+            onChange = { this.onChange }
```

Comment:
### Nitpick

This and the line below is not properly indented. Please check it out.